cypress/e2e/RoundTrips/DrawEditor.feature

```diff
@@ -0,0 +1,264 @@
+Feature: Test roundtrip of the application: draw editor
```

Comment:
I don't see the content of these files added in the RoundTrip :
- plugin/ArrangeComponentsPosition.feature
- plugin/DisplayComponentVariables.feature
- plugin/RenameComponentId.feature
- DeleteComponent.feature

Reply:
Done